wiki/codeReview.md

```diff
@@ -0,0 +1,104 @@
+| Reviewer                   | The reviewer, who is the person responsible for examining the code and reporting the results to the author. |
+
+PR (pull request)
+
+## The main code review rules
+
+- We are the team, and it means that we have the same aim - a successful project. That's why a code review is a place for dialog
+  and cooperation, not for arguing.
+- We have to be polite and calm during the review. And we have to remember always, that we are assessing only code, not a person
+  who has provided this code for review. That means we have to avoid different personal judgments and biases during the review.
+- If there is still a conflict, you have to remember that sometimes giving up and accepting mistakes are not a failure and
+  will make you and the project better.
+
+## Why should I make the code review? 
+
+- It helps our codebase to be cleaner, more stable, and more maintainable
+- It helps to support code standards that are applied to project
+- it helps developers share their knowledge about the project with each other
+- It makes the style of writing different parts of the system more common and consistent
+- It is a good self-development approach for developers
+- It facilitates new developers' onboarding process
+- It helps to avoid some bugs before it affects somebody
+
+## When should I take PR for review?
+
+- It is good practice taking reviews at a particular time every working day. For example, the reviewer can take PR for review every morning before the main part of work starts. Some tasks need to be reviewed urgently. It is ok to ask teammates to make such a review ASAP
+- It strongly requires every developer to be involved in the code review process
+- If the reviewer were assigned on review he has to make it. No exceptions to it
+- Reviewer shouldn't spend more than 60 minutes per day on the review
+
+## Responsibilities of maintainers of process
+
+### Developer have to:
+
+- The main responsibility of a Developer is provisioning an implementation according to the DoD items list
+- Follow code styles and accepted approaches
+- Follow the requirements for PR
+- Be polite to the reviewer
+- Try to understand comments and fix issues, or provide strongly reasoned objections why they shouldn't be fixed
+
+### Reviewer have to:
+
+- Briefly check and validate if provided code matches with DoD
+- Check if provided code matches with project Code Style and accepted project practice. Even better if the reviewer has a code review checklist (Provided bellow)
+- Be polite to a developer
+- Remember that reviewer evaluates the code, not the person who wrote this code
+- Add sensibly and reasoned comments about some issues in the code. It will be best if the reviewer provides some decisions for noticed issues
+- Check fixes when the developer will make them
+- Switch to reviewing branch and make a short smoke test (recommended)
+- Approve PR if it looks good to you
+
+## Requirements for PR
+
```

Comment:
@EvgeniyKvasyuk  Lets add a PR template with the requirements checklist?

Reply:
sounds sensible, will do it